xrpl/models/transactions/clawback.py

```diff
@@ -0,0 +1,29 @@
+"""Model for Clawback transaction type and related flags."""
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+
+from xrpl.models.amounts import IssuedCurrencyAmount
+from xrpl.models.required import REQUIRED
+from xrpl.models.transactions.transaction import Transaction
+from xrpl.models.transactions.types import TransactionType
+from xrpl.models.utils import require_kwargs_on_init
+
+
+@require_kwargs_on_init
+@dataclass(frozen=True)
+class Clawback(Transaction):
+    """The clawback transaction claws back issued funds from token holders."""
+
+    amount: IssuedCurrencyAmount = REQUIRED  # type: ignore
```

Comment:
Shouldn't this tx also have an error check to ensure the issuer isn't the same as the account, like in xrpl.js?